scripts/assemble_changelog.py

```diff
@@ -219,8 +219,10 @@ def add_categories_from_text(self, filename, line_offset,
                                        category.name.decode('utf8'))
 
             body_split = category.body.splitlines()
+            re_has_url = re.compile('.*http[s]?://.*')
             for line_number, line in enumerate(body_split, 1):
-                if len(line) > MAX_LINE_LENGTH:
+                if not re_has_url.match(line.decode('utf-8')) and \
```

Comment:
Simpler (and microscopically faster): make a bytes regex and don't call `decode`.